test/test.js

```diff
@@ -0,0 +1,132 @@
+const chai = require("chai"),
```

Comment:
You should create a test file for each module instead of writing all tests in a single file.

Reply:
sure I will